### PR TITLE
s234

### PR DESCRIPTION
**Pull request by thanrobinson**
added extirpated range, polygons over extirpated become restoration, no matter probability or effort.

### CODE REVIEW

File: src/task_scl_eff_pot_hab.py [hunk 1]
```diff
@@ -44,8 +49,8 @@ class SCLPolygons(SCLTask):
         },
         "density": {
             "ee_type": SCLTask.FEATURECOLLECTION,
-            "ee_path": "projects/SCL/v1/Panthera_tigris/source/density/tiger_obs_density",
-            "static": True,
+            "ee_path": "projects/SCL/v1/Panthera_tigris/biome_density",
+            "maxage": 10,
```
Comment (saanobhaai) [inline src/task_scl_eff_pot_hab.py]: No `maxage` checking for featurecollections (yet) so I'd remove this

File: src/task_scl_eff_pot_hab.py [hunk 2]
```diff
@@ -289,16 +310,23 @@ def calc(self):
...
         survey_effort = ee.Image.constant(1)
 
         scl_polys = (
             ee.Image(1)
             .updateMask(scl_polys)
-            .addBands(scl_polys)
-            .addBands(probability_thresh)
-            .addBands(survey_effort)
-            .rename(["scl_poly", "size", "probability", "effort"])
+            .addBands([scl_polys, range_binary, probability_thresh, survey_effort])
```
Comment (saanobhaai) [inline src/task_scl_eff_pot_hab.py:327]: Is `size` missing?

**Review by saanobhaai — CHANGES_REQUESTED** [review]

File: src/task_scl_eff_pot_hab.py (hunk 1 above)
Comment (thanrobinson) [reply src/task_scl_eff_pot_hab.py]: done

File: src/task_scl_eff_pot_hab.py (hunk 2 above)
Comment (thanrobinson) [reply src/task_scl_eff_pot_hab.py:327]: good catch

**Pull request merged**